recipes-devtools/dlpack/dlpack_0.7.bb

```diff
@@ -0,0 +1,16 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=f62d4e85ba68a1574b74d97ab8dea9ab"
+
+SRC_URI = "git://github.com/dmlc/dlpack.git;protocol=https;branch=main"
+
+SRCREV = "e2bdd3bee8cb6501558042633fa59144cc8b7f5f"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+PACKAGECONFIG = ""
+PACKAGECONFIG[docs] = "-DBUILD_DOCS=ON,-DBUILD_DOCS=OFF"
+PACKAGECONFIG[mock] = "-DBUILD_MOCK=ON,-DBUILD_MOCK=OFF"
+
+ALLOW_EMPTY_${PN} = "1"
```

Comment:
I think you should remove this line

Reply:
@ichergui This is header only library. It will fails without this line during the install task.